dev/com.ibm.ws.security.wim.adapter.ldap/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -114,10 +114,10 @@ failoverServer.port.desc=LDAP failover server port.
 #==== LDAP FILTER CONFIGURATION =====
 
 userFilter=User filter
-userFilter.desc=An LDAP filter clause for searching the user registry for users.
+userFilter.desc=An LDAP filter clause for searching the user registry for users. When defined, this filter requires an Attribute Value Assertion (AVA) containing a %v. For example, uid=%v. The %v in the AVA will be replaced during searches with the user or user pattern being searched for.  
```

Comment:
Change,
"The %v in the AVA will be replaced during searches with the user or user pattern being searched for."
to,
"During searches, the %v in the AVA is replaced with the user or user pattern being searched for."

Reply:
Made the changes.